components/aws/sagemaker/tests/integration_tests/utils/argo_utils.py

```diff
@@ -2,7 +2,12 @@
 
 
 def print_workflow_logs(workflow_name):
-    output = utils.run_command(
-        f"argo logs {workflow_name} -n {utils.get_kfp_namespace()}"
-    )
+    output = get_workflow_logs(workflow_name)
```

Comment:
nit: Ideally there should be some error handling here if workflow name is not found, but I guess argo will take care of the right error.

Reply:
What is the response to `get_workflow_logs` if there is a bad input? I imagine it would be a garbled error message that we are then using as the name.